frontend/src/libraries/mixpanel.ts

```diff
@@ -0,0 +1,48 @@
+import mixpanel from "mixpanel-browser";
+
+const BROWSER_MIXPANEL_TOKEN = "7cc6cbaab18d00de1b06ca9b4e773ed7";
+
+if (process.env.NEAR_EXPLORER_DATA_SOURCE) {
```

Comment:
Since `process` is not available in a browser context, I believe this is going to be never `true` on the browser side.

Reply:
we have this variable on the render environment file and this works for tracking